voting_metadata.go

```diff
@@ -0,0 +1,52 @@
+package pbft
+
+// NewVotingMetadata initializes instance of VotingMetadata
+func NewVotingMetadata(votingPower map[NodeID]uint64) *VotingMetadata {
```

Comment:
This data structure got weirder now. We return constant values, that are set inside the struct so they are not parametrizable and `CalculateVotingPower` is fixed. To me, this `Metadata` is just a struct with constant values.